langchain_dappier/retrievers.py

```
@@ -0,0 +1,212 @@
+"""Dappier retrievers."""
+
+import os
+from typing import Any, List, Literal, Optional
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)